server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/DownloadGet.feature

```diff
@@ -80,4 +81,22 @@ Feature: Download GET
     And a connected user "username1@domain.tld"
     And "username1@domain.tld" has a mailbox "INBOX"
     When "username1@domain.tld" downloads "1"
-    Then the user should receive a not found response
\ No newline at end of file
+    Then the user should receive a not found response
+
+  Scenario: User cannot download attachment of another user when shared mailbox
+    Given "username@domain.tld" mailbox "sharedMailbox" contains a message "1" with an attachment "2"
+    And "username@domain.tld" shares its mailbox "sharedMailbox" with "username1@domain.tld"
+    And a connected user "username1@domain.tld"
+    And "username1@domain.tld" has a mailbox "sharedMailbox"
+    When "username1@domain.tld" downloads "2"
+    Then the user should receive that blob
+    And the blob size is 3071
+
+  Scenario: User cannot download message blob of another user when shared mailbox
```

Comment:
please add a fixup s/cannot/can to ed7f58342d292fcd4311ba7ff8d6b18fa51475f3

Reply:
please add following scenarios:
Scenario: User cannot download attachment of another user when shared mailbox to someone else
Scenario: User cannot download message blob of another user when shared mailbox to someone else